src/geolocation/component.js

```diff
@@ -162,20 +182,20 @@ function Controller(
         let msg;
         switch (error.code) {
           case 1:
-            msg = gettextCatalog.getString('User denied the request for Geolocation.');
+            msg = this.gettextCatalog_.getString('User denied the request for Geolocation.');
```

Comment:
This will not work for the string collection...

Reply:
The string collection only works with ``gettextCatalog``?
This variable is not available outside of the constructor... Unless if we create an alias  this way:
```
const gettextCatalog = this.gettextCatalog_;
```
Do I really have to do so?

Reply:
> The string collection only works with gettextCatalog?

Yes

> Do I really have to do so?

Yes